test/functional/test_framework/util.py

```diff
@@ -308,6 +308,15 @@ def get_bip9_status(node, key):
     info = node.getblockchaininfo()
     return info['bip9_softforks'][key]
 
+def get_buried_softforks_status(node, key):
```

Comment:
nit: prefer a name like `buried_softfork_active()`, since this is returning a bool (True if the softfork is active)